.github/workflows/test-template.yml

```diff
@@ -68,7 +68,7 @@ jobs:
 
       - uses: pnpm/action-setup@v2.2.4
         with:
-          version: 7
+          version: 8
```

Comment:
Does not to change, just a suggestion: maybe we could move this value to the `inputs` and update it in a single place next time?